docs/UserGuide.md

```diff
@@ -177,99 +178,23 @@ A student can have any number of social media links. Add multiple social media l
 
 #### Acceptable Values:
 * Name: Full names with alphabetical characters. Maximum 30 characters.
-* Major: Valid major names at NUS. More information listed [below](#valid-majors)
-* Year: Numeric year level. 
-* Email: Valid email address ending in “@u.nus.edu”. Maximum 20 characters.
+* Major: Valid major names at NUS. View the [appendix](#valid-majors) for more information.
+* Year: Numeric year level. Accept values between `1` and `6` inclusive.
+* Email: Valid email address ending in `@u.nus.edu`. Maximum 20 characters.
 * Description: Maximum 150 characters.
-* Tutorials: Two digit integers between 01 and 22 inclusive. Multiple tutorial slots can be added by using `t/` repeatedly.
+* Tutorials: Two digit integers between `01` and `22` inclusive. Multiple tutorial slots can be added by using `t/` repeatedly.
 * Social Media Link: Valid URL format to social media account. Multiple URLs can be added by using `sm/` repeatedly.
```

Comment:
Maybe can mention starting with http:// to be clear

Reply:
it's https://

Reply:
edited